jax_unirep/evotuning.py

```diff
@@ -160,10 +161,122 @@ def length_batch_input_outputs(
     return xs, ys
 
 
+def fit_length_batches(
+    params: Dict, state, update, get_params, xs, ys, step_size: float = 0.001,
```

Comment:
Can we get type annotations on this function? I'm having a bit of trouble picturing in my head what data types are what.